client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -306,56 +294,62 @@ class DomainItem extends PureComponent {
 		const { translate } = this.props;
 
 		if ( [ domainTypes.MAPPED, domainTypes.REGISTERED ].indexOf( domainDetails.type ) === -1 ) {
-			return this.renderMinus();
+			return;
 		}
 
 		if ( hasGSuiteWithUs( domainDetails ) ) {
 			const gSuiteMailboxCount = getGSuiteMailboxCount( domainDetails );
 
-			return translate( '%(gSuiteMailboxCount)d mailbox', '%(gSuiteMailboxCount)d mailboxes', {
-				count: gSuiteMailboxCount,
-				args: {
-					gSuiteMailboxCount,
-				},
-				comment: 'The number of GSuite mailboxes active for the current domain',
-			} );
+			const text = translate(
+				'%(gSuiteMailboxCount)d mailbox',
+				'%(gSuiteMailboxCount)d mailboxes',
+				{
+					count: gSuiteMailboxCount,
+					args: {
+						gSuiteMailboxCount,
+					},
+					comment: 'The number of GSuite mailboxes active for the current domain',
+				}
+			);
+			return <span className="domain-item__meta-item">{ text }</span>;
 		}
 
 		if ( hasTitanMailWithUs( domainDetails ) ) {
 			const titanMailboxCount = getMaxTitanMailboxCount( domainDetails );
 
-			return translate( '%(titanMailboxCount)d mailbox', '%(titanMailboxCount)d mailboxes', {
+			const text = translate( '%(titanMailboxCount)d mailbox', '%(titanMailboxCount)d mailboxes', {
 				args: {
 					titanMailboxCount,
 				},
 				count: titanMailboxCount,
 				comment: '%(titanMailboxCount)d is the number of mailboxes for the current domain',
 			} );
+			return <span className="domain-item__meta-item">{ text }</span>;
 		}
 
 		if ( hasEmailForwards( domainDetails ) ) {
 			const emailForwardsCount = getEmailForwardsCount( domainDetails );
 
-			return translate( '%(emailForwardsCount)d forward', '%(emailForwardsCount)d forwards', {
+			const text = translate( '%(emailForwardsCount)d forward', '%(emailForwardsCount)d forwards', {
 				count: emailForwardsCount,
 				args: {
 					emailForwardsCount,
 				},
 				comment: 'The number of email forwards active for the current domain',
 			} );
+			return <span className="domain-item__meta-item">{ text }</span>;
 		}
 
 		if ( ! canCurrentUserAddEmail( domainDetails ) ) {
-			return this.renderMinus();
+			return;
 		}
 
 		return (
-			<Button compact onClick={ this.addEmailClick }>
-				{ translate( 'Add', {
-					context: 'Button label',
-					comment: '"Add" as in "Add an email"',
-				} ) }
-			</Button>
+			<span className="domain-item__meta-item">
+				<Button compact onClick={ this.addEmailClick } className="domain-item__email-button">
+					{ translate( 'Add email', { context: 'Button label' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Add email' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-item-list-updates)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.